### PR TITLE
[docs] PR for Document PLAT-1962 feature (DocPLAT-2304)

### PR DESCRIPTION
**Pull request by lizayugabyte**
Ticket: https://yugabyte.atlassian.net/browse/PLAT-2304
Preview: https://deploy-preview-10627--infallible-bardeen-164bc9.netlify.app/latest/yugabyte-platform/security/create-kms-config.md/aws-kms/

### CODE REVIEW

File: docs/content/latest/yugabyte-platform/security/enable-encryption-at-rest.md [hunk 1]
```diff
@@ -33,54 +33,56 @@ For details on the features, assumptions, design, data key management, universe
...
 
 1. Open the Yugabyte Platform console and click **Universes**.
 
 2. Select the universe for which to enable encryption.
 
-3. Select **Edit Universe > Edit Security > Encryption-at-Rest**. 
+3. Select **Actions > Edit Universe > Edit Security > Encryption-at-Rest**. 
 
-4. Deselect **Enable Encryption-at-Rest for <UNIVERSE_NAME>** and click **Submit**.
+4. In the **Manage Encryption at-Rest** dialog, toggle **Enable Encryption-at-Rest for <Universe_name>**.
```
Comment (ddhodge) [inline docs/content/latest/yugabyte-platform/security/enable-encryption-at-rest.md:77]: "Manage Encryption at-Rest"? remove both hyphens? or neither?

File: docs/content/latest/yugabyte-platform/security/enable-encryption-at-rest.md [hunk 2]
```diff
@@ -33,54 +33,56 @@ For details on the features, assumptions, design, data key management, universe
...
    When the encryption is enabled, the **Key Management Service Config** option appears.
 
 5. Select your KSM configuration from the **Key Management Service Config** list. The list displays only preconfigured KMS configurations. If you need to create one, see [Create a KMS configuration](../create-kms-config).
 
-6. Verify that the `EnableEncryptionAtRest` task completed successfully.
+6. Click **Submit**.
+
+7. Verify that the `EnableEncryptionAtRest` task completed successfully.
 
 ## Back up and restore data from an encrypted at rest universe
```
Comment (ddhodge) [inline docs/content/latest/yugabyte-platform/security/enable-encryption-at-rest.md:88]: add "Select Actions > Edit Universe" to steps in Rotate and Disable sections?
also missing "In the Manage Encryption at-Rest dialog, " in step 3 of disable section?

**Review by ddhodge — COMMENTED** [review]
Format problem, minor fixes

File: docs/content/latest/yugabyte-platform/security/create-kms-config.md/aws-kms.md [hunk 3]
```diff
@@ -47,22 +47,22 @@ The AWS user associated with the a KMS configuration requires the following mini
```
[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/latest/yugabyte-platform/security/create-kms-config.md/aws-kms.md:47]: Fixed

File: docs/content/latest/yugabyte-platform/security/create-kms-config.md/aws-kms.md [hunk 4]
```diff
@@ -47,22 +47,22 @@ The AWS user associated with the a KMS configuration requires the following mini
 
 {{< /note >}}
 
-To create a KMS configuration that uses AWS Key Management Service (KMS):
+You can create a KMS configuration that uses AWS Key Management Service (KMS) as follows:
```
[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/latest/yugabyte-platform/security/create-kms-config.md/aws-kms.md:51]: Fixed

File: docs/content/latest/yugabyte-platform/security/create-kms-config.md/aws-kms.md [hunk 5]
```diff
@@ -47,22 +47,22 @@ The AWS user associated with the a KMS configuration requires the following mini
...
+    - **Configuration Name** — Enter a meaningful name for your configuration.
+    - **KMS Provider** — Select **AWS KMS**.
+    - **Use IAM Profile** — Specify whether or not to use an IAM profile attached to an EC2 instance running YugabyteDB. For more information, see [Using instance profiles](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_use_switch-role-ec2_instance-profiles.html).
     - **Access Key Id** — Enter the identifier for the access key.
     - **Secret Key Id** — Enter the identifier for the secret key.
-    - **Region** — Select the AWS region where the CMK used to generate universe keys will be located. **Note:** This does not need to match the region where the encrypted universe resides on Amazon Web Services (AWS).
+    - **Region** — Select the AWS region where the CMK used to generate universe keys will be located. This setting does not need to match the region where the encrypted universe resides on Amazon Web Services (AWS).
```
[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/latest/yugabyte-platform/security/create-kms-config.md/aws-kms.md:61]: Fixed

File: docs/content/latest/yugabyte-platform/security/create-kms-config.md/aws-kms.md [hunk 6]
```diff
@@ -47,22 +47,22 @@ The AWS user associated with the a KMS configuration requires the following mini
...
     - **Customer Master Key ID** — Enter the identifier for the customer master key. If an identifier is not entered, a CMK ID will be auto-generated.
```
[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/latest/yugabyte-platform/security/create-kms-config.md/aws-kms.md:62]: Fixed

File: docs/content/latest/yugabyte-platform/security/create-kms-config.md/aws-kms.md [hunk 7]
```diff
@@ -36,7 +36,7 @@ Encryption at rest uses universe keys to encrypt and decrypt universe data keys.
```
Comment (ddhodge) [inline docs/content/latest/yugabyte-platform/security/create-kms-config.md/aws-kms.md:36]: KMS defined twice in paragraph above

File: docs/content/latest/yugabyte-platform/security/create-kms-config.md/aws-kms.md [hunk 8]
```diff
@@ -89,9 +89,8 @@ To create a KMS configuration that uses AWS Key Management Service (KMS):
             }
         ]
     }
-
-    ```
+```
```
Comment (ddhodge) [inline docs/content/latest/yugabyte-platform/security/create-kms-config.md/aws-kms.md:93]: Numbering in preview shows 1 2 3 4 6 7

File: docs/content/latest/yugabyte-platform/security/enable-encryption-at-rest.md [hunk 9]
```diff
@@ -33,54 +33,56 @@ For details on the features, assumptions, design, data key management, universe
...
-To enable encryption at rest during universe creation:
+You can enable encryption at rest during the universe creation as follows:
 
-1. Open the Yugabyte Platform console and click **Universes**.
+1. Open the Yugabyte Platform console and navigate to **Universes**.
 2. Click **Create Universe**. The **Universe Configuration** page appears.
 3. After you select a provider, the **Instance Configuration** section expands to show more options.
 4. Select the **Enable Encryption at Rest** option. The **Key Management Service Config** option appears.
-5. Select your KMS configuration from the **Key Management Service Config** drop-down list. The list displays only preconfigured KMS configurations. If you need to create one, see [Create a KMS configuration](../create-kms-config).
+5. Select your KMS configuration from the **Key Management Service Config** list. The list displays only preconfigured KMS configurations. If you need to create one, see [Create a KMS configuration using AWS KMS](../create-kms-config.md/aws-kms/) and [Create a KMS configuration using AWS KMS](../create-kms-config.md/equinix-smartkey/).
```
Comment (ddhodge) [inline docs/content/latest/yugabyte-platform/security/enable-encryption-at-rest.md:42]: title of second link wrong

File: docs/content/latest/yugabyte-platform/security/enable-encryption-at-rest.md [hunk 10]
```diff
@@ -33,54 +33,56 @@ For details on the features, assumptions, design, data key management, universe
...
-3. Select **Edit Universe > Edit Security > Encryption-at-Rest**. 
+3. Select **Actions > Edit Universe > Edit Security > Encryption-at-Rest**. 
 
-4. Deselect **Enable Encryption-at-Rest for <UNIVERSE_NAME>** and click **Submit**.
+4. In the **Manage Encryption at Rest** dialog, toggle **Enable Encryption at Rest for <universe_name>**.
 
    When the encryption is enabled, the **Key Management Service Config** option appears.
 
-5. Select your KSM configuration from the **Key Management Service Config** list. The list displays only preconfigured KMS configurations. If you need to create one, see [Create a KMS configuration](../create-kms-config).
+5. Select your KSM configuration from the **Key Management Service Config** list. The list displays only preconfigured KMS configurations. If you need to create one, see [Create a KMS configuration using AWS KMS](../create-kms-config.md/aws-kms/) and [Create a KMS configuration using AWS KMS](../create-kms-config.md/equinix-smartkey/).
```
Comment (ddhodge) [inline docs/content/latest/yugabyte-platform/security/enable-encryption-at-rest.md:81]: title of second link wrong

**Review by ddhodge — APPROVED** [review]
LGTM with a few minor fixes

**Pull request merged**